Dockerfile.dockerignore

```diff
@@ -3,6 +3,7 @@
 # must-have toplevel files
 !/go.sum
 !/go.mod
+!/Makefile
```

Comment:
Adding this makes the docker cache invalidated on changes to the makefile. That should not happen as it does not affect the built binary.